cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -341,13 +342,16 @@ define(["jquery", "js/common_helpers/ajax_helpers", "js/views/utils/view_utils",
                 it('can start reindex of a course - respond fail', function() {
```

Comment:
This isn't strictly part of this change, but I don't like the names of these tests as they don't read as English. I'd make it something like:

    it('shows an error message when reindexing fails', ...